### PR TITLE
Add MO to PYTHONPATH in lin-debian Azure job

### PR DESCRIPTION
**Pull request by p-wysocki**
### Details:
 - Fix for a failing postcommit check

### Tickets:
 - N/A

### CODE REVIEW

File: .ci/azure/linux_debian.yml [hunk 1]
```diff
@@ -360,7 +360,7 @@ jobs:
       # - tests install dir for mock_py
       LD_LIBRARY_PATH: $(INSTALL_TEST_DIR)
       # For python imports to import pybind_mock_frontend
-      PYTHONPATH: $(INSTALL_TEST_DIR)
+      PYTHONPATH: $(INSTALL_TEST_DIR):$(REPO_DIR)/tools/ovc/
```
Comment (ilya-lavrenov) [inline .ci/azure/linux_debian.yml:363]: tools/ovc should be available after sourcing `setupvars.sh`

[the remark this answers is not in the document]
Comment (akuporos) [reply .ci/azure/linux_debian.yml]: I vote to change, also less refactoring in the future

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**